models/sipnet/R/read.restart.SIPNET.R

```diff
@@ -4,70 +4,65 @@
 ##' 
 ##' @param outdir      output directory
 ##' @param runid       run ID
-##' @param time        year that is being read
+##' @param stop.time        year that is being read
 ##' @param settings    PEcAn settings object
-##' @param variables   variables to be extracted
+##' @param var.names   var.names to be extracted
 ##' 
 ##' @description Read Restart for SIPNET
 ##' 
 ##' @return X.vec      vector of forecasts
 ##' @export
 ##' 
-read.restart.SIPNET <- function(outdir,runid,time,settings,variables,sample_parameters=NULL){
+read.restart.SIPNET <- function(outdir,runid,stop.time,settings,var.names,params){
 
-  if(sample_parameters == TRUE){
-    load(file.path(settings$outdir, "samples.Rdata"))
-    prior.sla <- mean(ensemble.samples[[which(names(ensemble.samples)!='soil')[1]]]$SLA) #HACK
-  }else{
-    load(file.path(settings$outdir, paste0("ensemble.samples.",settings$state.data.assimilation$prior,".Rdata")))
-    prior.sla <- ens.samples[[which(names(ensemble.samples)!='soil')[1]]]$SLA
-  }
+  prior.sla <- params[[which(names(ensemble.samples)!='soil')[1]]]$SLA
   
   forecast <- list()
   
   #Read ensemble output
   ens <- read.output(runid = runid,outdir = file.path(outdir, runid),
-                     start.year = time, end.year=time,
-                     variables=variables)
+                     start.year = strftime(stop.time, '%Y'),
```

Comment:
Does the `stop.time` argument passed to both `read.restart.LINKAGES` and `read.restart.SIPNET` have the same format or being used in the same format in `read.output`? because in `read.restart.LINKAGES` you directly use `stop.time`, whereas here you exract `year` by `strftime(stop.time, '%Y')`. Just wanted to note